apps/server/test/lexical/server/code_intelligence/completion/translations/macro_test.exs

```diff
@@ -612,4 +612,50 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.MacroTest do
       refute boosted?(completion)
     end
   end
+
+  test "test has three completions", %{project: project} do
+    assert {:ok, completions} =
+             project
+             |> complete(inside_exunit_context("test|"))
+             |> fetch_completion("test ")
+
+    assert [%Completion.Item{}, %Completion.Item{}, %Completion.Item{}] = completions
+  end
+
+  test "test completion snippets", %{project: project} do
+    assert {:ok, completions} =
+             project
+             |> complete(inside_exunit_context("test|"))
+             |> fetch_completion("test ")
+
+    assert %Completion.Item{
+             label: "test (Define an empty test/1)",
+             detail: "",
+             insert_text_format: :snippet,
+             insert_text: "test \"${0:name}\"\n"
+           } = get_in(completions, [Access.at(0)])
```

Comment:
I really don't like assertions like this; If something goes wrong, you have to parse a diff. I much prefer assertions like:
```elixir
assert item.label == "test ..."
assert item.insert_text_format == "snippet"
```
Additionally, doing assertions on the snippets is prescriptive; we have an `apply_completion` function, so you can ensure the outcome is what you expect

Reply:
> If something goes wrong, you have to parse a diff

Oh such a good point